crates/cubecl/Cargo.toml

```diff
@@ -19,26 +19,25 @@ default = [
     "cubecl-wgpu?/default",
     "cubecl-cuda?/default",
 ]
-linalg = ["dep:cubecl-linalg"]
 exclusive-memory-only = [
     "cubecl-wgpu?/exclusive-memory-only",
     "cubecl-runtime/exclusive-memory-only",
 ]
+linalg = ["dep:cubecl-linalg"]
 std = ["cubecl-core/std", "cubecl-wgpu?/std", "cubecl-cuda?/std"]
 template = ["cubecl-core/template"]
 
 # Runtimes
 cuda = ["cubecl-cuda"]
 wgpu = ["cubecl-wgpu"]
-wgpu-spirv = ["cubecl-wgpu-spirv"]
+wgpu-spirv = ["cubecl-wgpu/spirv"]
```

Comment:
I would put `["wgpu", "cubecl-wgpu/spirv"]` so that the wgpu feature flag is activated.